sdks/python/apache_beam/pipeline.py

```diff
@@ -131,8 +131,7 @@ class Pipeline(HasDisplayData):
   (e.g. ``input | "label" >> my_transform``).
   """
   @classmethod
-  def runner_implemented_transforms(cls):
-    # type: () -> FrozenSet[str]
+  def runner_implemented_transforms(cls) -> FrozenSet[str]:
```

Comment:
Bit of a larger ask, but let's migrate these type hints away from the typing module to PEP-585 versions (https://peps.python.org/pep-0585/)

e.g. the built-in `dict`, `list`, `tuple`,`set`, `frozenset` types, collections.abc for Iterable, Mapping, and Sequence